config.m4

```diff
@@ -115,7 +82,4 @@ if test "$PHP_SOLR" != "no"; then
     						 $ext_shared,, [$COVERAGE_CFLAGS])
     PHP_ADD_BUILD_DIR($abs_builddir/$subdir, 1)
     PHP_SUBST(SOLR_SHARED_LIBADD)
-  ], [
-    AC_MSG_ERROR([xml2-config not found. Please check your libxml2 installation.])
-  ])
```

Comment:
Already raised by  PHP_SETUP_LIBXML

Reply:
I see, so PHP_SETUP_LIBXML, would do the check for the actual libxml2 ?

Reply:
yes, PHP_SETUP_LIBXML is enough